src/org/ods/component/CopyImageStage.groovy

```diff
@@ -0,0 +1,90 @@
+package org.ods.component
+
+import groovy.transform.TypeChecked
+import groovy.transform.TypeCheckingMode
+import org.ods.services.OpenShiftService
+import org.ods.util.ILogger
+
+@SuppressWarnings('ParameterCount')
+@TypeChecked
+class CopyImageStage extends Stage {
+
+    public final String STAGE_NAME = 'Copy Container Image'
+    private final OpenShiftService openShift
+    private final ILogger logger
+    private final CopyImageOptions options
+
+    @TypeChecked(TypeCheckingMode.SKIP)
+    CopyImageStage(
+        def script,
+        IContext context,
+        Map<String, Object> config,
+        OpenShiftService openShift,
+        ILogger logger) {
+        super(script, context, logger)
+        this.options = new CopyImageOptions(config)
+        this.openShift = openShift
+        this.logger = logger
+    }
+
+    // This is called from Stage#execute if the branch being built is eligible.
+    @TypeChecked(TypeCheckingMode.SKIP)
+    protected run() {
+        logger.info("Copy the image ${options.sourceImageUrlIncludingRegistry}!")
+        final String STR_DOCKER_PROTOCOL = 'docker://'
+
+        logger.info("Resolved source Image data: " +
+            "${this.options.sourceImageUrlIncludingRegistry} into ${this.options},\n" +
+            "importing into: " +
+            "${STR_DOCKER_PROTOCOL}${context.clusterRegistryAddress}/${context.cdProject}/${this.options.image}")
+
+        // to make this work, we need a target image stream first
+        openShift.findOrCreateImageStream("${context.cdProject}", "${this.options.image.split(':').first()}")
+
+        // read the target registry auth (for this project's -cd project ..)
+        def targetInternalRegistryToken = steps.readFile '/run/secrets/kubernetes.io/serviceaccount/token'
+
+        def sourcetoken = options.sourceCredential ? "--src-creds ${options.sourceCredential}" : ''
+
+        try {
+            steps.sh """
+            set +x
+            skopeo copy --src-tls-verify=false ${sourcetoken} \
```

Comment:
`--src-tls-verify=false`?

Reply:
Please see https://github.com/opendevstack/ods-pipeline/blob/master/cmd/deploy-with-helm/main.go#L189-L218, this should be possible.